docs/dev-tools/painlesslab/index.asciidoc

```diff
@@ -0,0 +1,22 @@
+[role="xpack"]
+[[painlesslab]]
+== Painless Lab
+
+beta::[]
+
+The Painless Lab is an interactive code editor that lets you test and
+debug Painless scripts in real-time. You can use the Painless scripting
```

Comment:
Can we make "Painless" a link to https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-painless.html ?